index.js

```diff
@@ -339,16 +352,140 @@ class Raspi extends events.EventEmitter {
     }
   }
 
+  i2cConfig(delay) {
+    if (this[i2c] === undefined) {
```

Comment:
Remember, ```this.alive``` is only true until the peripheral is destroyed (https://github.com/bryan-m-hughes/raspi-peripheral/blob/master/index.js#L47). This is exactly what we want to happen, because as soon as someone does ```board.pinMode('SCA0', OUTPUT)```, we want I2C to be disabled from then on out. We can only do that if we instantiate on startup.

Also, as we discussed in #16, we decided to drop support for baudRate for the time being anyways, so you can remove the ```execSync``` calls in their entirety.